homeassistant/components/icloud/device_tracker.py

```diff
@@ -30,25 +29,45 @@
 
 async def async_setup_entry(
     hass: HomeAssistantType, entry: ConfigEntry, async_add_entities
-):
-    """Configure a dispatcher connection based on a config entry."""
-    username = entry.data[CONF_USERNAME]
+) -> None:
+    """Set up device tracker for iCloud component."""
+    account = hass.data[DOMAIN][entry.unique_id]
+    tracked = set()
+
+    @callback
+    def update_account():
+        """Update the values of the account."""
+        add_entities(account, async_add_entities, tracked)
```

Comment:
This needs to be `async_add_entities` ? 
```suggestion
        async_add_entities(account, async_add_entities, tracked)
```